enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -37,12 +40,23 @@ import "metabase/css/vendor.css";
 
 export interface MetabaseProviderProps {
   children: ReactNode;
-  config: SDKConfig;
+  authConfig: MetabaseAuthConfig;
   pluginsConfig?: SdkPluginsConfig;
   eventHandlers?: SdkEventHandlersConfig;
   theme?: MetabaseTheme;
   className?: string;
+
+  /** Defines the display language. Accepts an ISO language code such as `en` or `de`. Defaults to `en`. */
```

Comment:
[there's a bug](https://metaboat.slack.com/archives/C063Q3F1HPF/p1733503500527059?thread_ts=1733501696.469049&cid=C063Q3F1HPF) currently that the SDK doesn't support locale like `en-US`.

We should include such a format to provide more clarity once we fix the bug.

Reply:
@WiNloSt Good catch, let me note down that country code suffixes such as `en-US` is not supported.

Reply:
Clarified in https://github.com/metabase/metabase/pull/50585/commits/d35e5fc13122b010db91eabfefd74d8867ab8ad4